src/zope/interface/_zope_interface_coptimizations.c

```diff
@@ -759,7 +759,7 @@ lookup_traverse(lookup *self, visitproc visit, void *arg)
 
 static int
 lookup_clear(lookup *self)
-{
+{	
```

Comment:
I would prefer not to see unnecessary trailing whitespace being added to an unrelated function.

Reply:
I appreciate you removing it.

Reply:
@y-fujisaki2 I believe @mgedmin was requesting that you update the PR to avoid this problem. I am making the same request.